nnvm/python/nnvm/frontend/tensorflow.py

```diff
@@ -669,6 +669,22 @@ def _impl(inputs, in_state_c, in_state_h, attr, params):
     return _impl
 
 
+def _pad(name):
+    def _impl(inputs, attr, params):
+        padlist = params.pop(inputs[1].list_output_names()[0]).asnumpy()
+        paddings = tuple([tuple(l) for l in padlist])
+        attr['pad_width'] = paddings
+        attr['pad_value'] = 0
```

Comment:
For Pad constant_value should be taken from attributes (should be zero only if not available in attributes.).

Reply:
Pad doesn't have `constant_value` as attribute. Please check my previous comment.